imports.go

```diff
@@ -0,0 +1,7 @@
+//go:build tinygo.wasm
+
+package httpwasm
+
+//go:wasm-module httpwasm
```

Comment:
Since it's already wasm, maybe it should be just `http`?

Reply:
very good idea